src/main/java/org/khaleesi/carfield/tools/sparkjobserver/api/SparkJobServerClientImpl.java

```diff
@@ -1,16 +1,12 @@
 package org.khaleesi.carfield.tools.sparkjobserver.api;
 
-import java.io.BufferedInputStream;
-import java.io.Closeable;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileNotFoundException;
-import java.io.IOException;
-import java.io.InputStream;
+import java.io.*;
```

Comment:
xx.xxx.* is not a good way to import class, please use the explicit xxx.xxxx.YYY import sentence instead.
Using Ctrl+shift+o in eclipse jdt can do it for you.